File: 1683-maximum-number-of-coins-you-can-get/maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        steps = len(piles) // 3
        piles.sort(reverse=True)
        print(piles)
        i = 1
        sum = piles[1]
        while i < len(piles):
            i += 2
            if i < len(piles): sum += piles[i]
            if i+1 == len(piles) - steps: break
        return sum

File: 1683-maximum-number-of-coins-you-can-get/maximum-number-of-coins-you-can-get.cpp
class Solution {
public:
    int maxCoins(vector<int>& piles) {
        int steps = piles.size() / 3;
        sort(piles.rbegin(), piles.rend());
        int sum = 0;
        for (int i = 1; i < piles.size() - steps; i+=2)
        {
            sum += piles[i];
        }
        return sum;
    }
};